docs/blockstates/extendedStates.md

```diff
@@ -0,0 +1,82 @@
+Extended States
+===================
+
+Prerequisites:
+* Understanding of [block states](./states.md). This article will assume you have read and understood the linked article.
+* Limited understanding of custom baked models, as they are pretty much the only use case for extended states.
+
+Motivation
+------------
+
+The rendering system of Minecraft 1.8 and above operates exclusively upon `IBlockState` instances, in that it assumes all information about the block's appearance is captured in that single object. The problem, however, is that regular `IBlockState`s need to know all possible combinations of properties and values ahead of time. This becomes a problem if you do not know all possible values at startup, or if the number of possible values is very large or infinite.
+
+*Extended States* and *Unlisted Properties* are Forge-introduced extensions to the blockstates system that address this problem.
+
+!!! Note
+	Extended states and unlisted properties are systems intended to address this problem only in the context of rendering.
+	Although the logical server still has the code and infrastructure for extended states, it cannot communicate it over the network, and nearly all server-side logical actions disregard extended states completely.
+	As a result, extended states are intended to only be used on the logical client.
```

Comment:
See https://github.com/MinecraftForge/MinecraftForge/pull/1518#issuecomment-66521210.

This should be made less prescriptive about the usage, I think.

Reply:
How about I just drop the 1st and 3rd sentences and keep the 2nd?

Reply:
That sounds good, yes.

Reply:
As for the second, I don't know of any serverside logic that uses extended states. Theoretically a mod could but there's not really a reason to.